lib/urcli-money.js

```diff
@@ -66,8 +66,17 @@ function definePeriods () {
       }
       const yearAndMonth = arg < 9 ? `${year}-0${arg}` : `${year}-${arg}`
       let start = moment(yearAndMonth).utc()
-      let end = moment(yearAndMonth).utc().add(1, 'M')
-      periods.push([start, end])
+      // Check if month asked is the current month
+      let currentMonth = moment.utc().add(1, 'M').month()
+      if (currentMonth == arg) {
+         // Make the last day to query as being today
+         let end = moment.utc()
```

Comment:
I've had trouble with the API not returning the latest reviews when I get the time very precisely like this. That is why I add one hour to `defaultEnd` up above. Here, you would not lose anything by adding time. You could simply go for midnight on today: `moment.utc().endOf('day')`.